<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addsensorwidth</class>
 <widget class="QDialog" name="Dialog_addsensorwidth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加相机参数</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_model">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>相机焦距：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>相机型号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_CANCEL">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_focus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>添加相机参数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>相机视场角(FOV)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fov">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
